doc/source/tutorials/pidloops.rst

```diff
@@ -309,7 +309,7 @@ The script we'll use to tune the highly overpowered rocket shown will launch the
 
     // feedback based on atmospheric efficiency
     LOCK surfspeed TO SHIP:VELOCITY:SURFACE:MAG.
-    LOCK atmoeff TO surfspeed / SHIP:TERMVELOCITY.
+    LOCK atmoeff TO surfspeed / SHIP:TERMVELOCITY.  //Using deprecated TERMVELOCITY syntax
```

Comment:
Well, darn.

Merely marking this as deprecated probably isn't good enough here.   Since this is a tutorial showing people how to use kOS, it should be re-written to not use this at all.  And that might mean having to go over the logic of this tutorial from scratch again.